globus_automate_client/cli/auth.py

```diff
@@ -68,12 +72,12 @@ def __init__(self, token_store: Union[pathlib.Path, str]):
         self.modified = False
 
     @property
-    def tokens_for_environment(self) -> Dict[str, TokenSet]:
+    def tokens_for_environment(self):
```

Comment:
Why is the annotation gone?

Reply:
Both of these lines were removed in 2f7bf211d6320fa05f3b64b1d06e2a01e40aacd4 because mypy _super_ dislikes these lines. No matter how specific I got, no matter how I changed it, mypy traced problems deeper into the code and to other usages.

Removing the lines resolved the problem in an incorrect way.

Reply:
I'm definitely open to revisiting this, but it will take some time to unravel the threads that mypy tugged at when I first tried fixing this...

Reply:
I tried to fix this again just now but mypy really doesn't like these annotations. I'm open to suggestions how to proceed!

Reply:
I see quite a lot of mypy errors either way—I'm not sure if maybe I don't have this set up right? As is maybe we can leave the annotation, and leave resolving the errors to a future effort on typing.

Reply:
@rudyardrichter I'm now able to reproduce the mypy errors. This may point to some issues between SDK v3 and the client unfortunately, because some of the errors are "such-and-such attribute doesn't exist on that class". I'll keep working on this.

Reply:
I think I've resolved all of these issues now. Unit tests are now passing.